README.md

```diff
@@ -97,6 +97,21 @@ const account = client.createKinAccount({
 ```
 See more about channels in the ["Channels" section](#Channels)
 
+You can retrieve the transaction history which is a list of your recent transactions: 
+
+Note: The data is available up to the 5 previous days from the moment of the transaction.
+```javascript
+const history = client.getTransactionHistory({address: keypair.publicAddress});
```

Comment:
follow the rest of code examples, which use promises and then block for demonstration.
use other params for demonstration (`limit` and `order`)